wikimedia/wikipedia-ios
# Image recommendations -  hide settings toggle

**Phabricator:** https://phabricator.wikimedia.org/T364038

### Notes
* Checks edit count before adding Suggested Edit to the explore feed settings toggles

### Test Steps
1. On an account with less than 50 edits (or logged out), go to explore feed settings
2. Should not see the Suggested Edits toggle
3. Repeat on an account with more than 50 edits, the toggle should appear
